subsys/bluetooth/controller/ll_sw/ull_adv_iso.c

```diff
@@ -635,7 +618,7 @@ static int init_reset(void)
 	return 0;
 }
 
-static inline struct ll_adv_iso_set *ull_adv_iso_get(uint8_t handle)
+static struct ll_adv_iso_set *iso_get(uint8_t handle)
```

Comment:
I actually liked the previous name better. 
Perhaps `adv_iso_get`? `iso_get` feels too generic for ISO broadcaster BIG handles.